templates/CRM/Contact/Form/Task/Map/OpenStreetMaps.tpl

```diff
@@ -32,7 +32,7 @@
 {/if}
 {assign var=defaultZoom value=12}
 {literal}
-<script src="http://openlayers.org/api/OpenLayers.js" type="text/javascript"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/2.13.1/OpenLayers.js" type="text/javascript"></script>
```

Comment:
Protocol-relative URLs can be [considered an anti-pattern](http://www.paulirish.com/2010/the-protocol-relative-url/) , as there are no circumstances under which retrieving from HTTPS is a problem.  Privacy organizations (EFF, ACLU, etc) universally recommend using HTTPS wherever possible, and major browser vendors (Google, Mozilla, etc) are making rumblings about deprecating unencrypted HTTP.  Using https here ensures that the OpenLayers script is not tampered with, and protects the privacy of people requesting the script (e.g. revealing which script on cdnjs.cloudflare.com is being retrieved).

In short: there's nothing in particular to gain from a protocol-relative URL, a bit of clarity to gain from specifying the protocol explicitly, and a teensy-tiny nod toward privacy to use https even when the parent site is hosted under http.